src/com/lilithsthrone/game/character/npc/submission/RebelBaseInsaneSurvivor.java

```diff
@@ -0,0 +1,249 @@
+import com.lilithsthrone.game.character.race.RaceStage;
+import com.lilithsthrone.game.character.race.Subspecies;
+import com.lilithsthrone.game.dialogue.DialogueNode;
+import com.lilithsthrone.game.dialogue.npcDialogue.submission.RebelBaseInsaneSurvivorDialogue;
+import com.lilithsthrone.game.dialogue.responses.Response;
+import com.lilithsthrone.game.dialogue.utils.UtilText;
+import com.lilithsthrone.game.inventory.CharacterInventory;
+import com.lilithsthrone.game.inventory.InventorySlot;
+import com.lilithsthrone.game.inventory.clothing.AbstractClothing;
+import com.lilithsthrone.game.inventory.clothing.ClothingType;
+import com.lilithsthrone.game.inventory.clothing.DisplacementType;
+import com.lilithsthrone.game.inventory.item.ItemType;
+import com.lilithsthrone.main.Main;
+import com.lilithsthrone.utils.Util;
+import com.lilithsthrone.utils.Util.Value;
+import com.lilithsthrone.utils.colours.PresetColour;
+import com.lilithsthrone.world.WorldType;
+import com.lilithsthrone.world.places.PlaceType;
+import java.time.Month;
+import java.util.List;
+
+
+/**
+ *
+ * @author dustland.crow
+ */
+public class RebelBaseInsaneSurvivor extends NPC {
+    
+    public static final int RECRUITMENT_YEAR = 1990;
+    
+    public RebelBaseInsaneSurvivor() {
+        this(Gender.M_P_MALE, false);
+    }
+    
+    public RebelBaseInsaneSurvivor(Gender gender) {
+        this(gender, false);
+    }
+    
+    public RebelBaseInsaneSurvivor(boolean isImported) {
+        this(Gender.M_P_MALE, isImported);
+    }
+    
+    public RebelBaseInsaneSurvivor(Gender gender, boolean isImported) {
+        super(isImported, null, null, "",
+				(Main.game.getDateNow().getYear() - RECRUITMENT_YEAR) + 18, Util.randomItemFrom(Month.values()), 1+Util.random.nextInt(25),
+				5,
+				gender, Subspecies.HUMAN, RaceStage.HUMAN,
+				new CharacterInventory(10), WorldType.REBEL_BASE, PlaceType.REBEL_BASE_SLEEPING_AREA, false);
+        
+        if(!isImported) {
+            this.setLocation(Main.game.getPlayer(), true);
+            
+            // Spawn with level between 12 and 15
+            setLevel(12 + Util.random.nextInt(4));
+            
+            setName(Name.getRandomTriplet(this.getRace()));
+            this.setPlayerKnowsName(false);
+            this.setGenericName("insane survivor");
+            this.setEssenceCount(500);
+            setDescription(UtilText.parse(this, "While apparently mostly human, [npc.name] has evidently spent a long time underground and has lost [npc.her] mind."));
+            
+            CharacterUtils.generateItemsInInventory(this);
+            this.addItem(Main.game.getItemGen().generateItem(ItemType.MUSHROOM), 10, false, false);
+        
+            initPerkTreeAndBackgroundPerks();
+            this.setStartingCombatMoves();
+            
+            // Nobody wants to see the gyrations of a living crack zombie
+            this.unequipMove("tease");
```

Comment:
This doesn't seem to work, tease moves are still generated.
To fix it, add this to override some methods:
    @Override
    public void setStartingCombatMoves() {
        this.clearEquippedMoves();
        this.equipMove("strike");
        this.equipMove("twin-strike");
    }

    @Override
    public float getMoveWeight(AbstractCombatMove move, List<GameCharacter> enemies, List<GameCharacter> allies) {
        // Death or Glory!
        if (move.getType() == CombatMoveType.TEASE || move.getType() == CombatMoveType.DEFEND) {
            return 0;
        } else {
            return super.getMoveWeight(move, enemies, allies);
        }
    }

Reply:
Fixed

Reply:
Although getMoveWeight does not accept AbstractCombatMove parameters, changed to CombatMove (as in Silence.java)